src/main/resources/react/src/styledTheme/index.js

```diff
@@ -0,0 +1,26 @@
+export const theme = {
+	colors: {
+		shadow: 'rgba(89, 102, 116, 0.1)',
+		colorPopup: '#FDFDFF',
+		'btn-blue-normal': 'hsl(195, 86%, 50%)', // -> #12b6ed
+		'btn-blue-hover': 'hsl(195, 120%, 50%)', // -> #00bfff
+		'btn-blue-active': 'hsl(205, 100%, 50%)', // -> #0094ff
+		'btn-green-normal': 'hsl(134, 45%, 48%)', // ->  #43b15d
+		'btn-green-hover': 'hsl(134, 45%, 43%)', // -> #3c9f53
+		'btn-green-active': 'hsl(134, 100%, 27%)', // -> #008a20
+		'colorError': '#ff4c4c', // -> #ff4c4c
+		colorGrey: '#8E8E8E', // -> #8E8E8E
+		'btn-gb-disabled': '#DADADA', // -> #DADADA
+		'right-color-text': '#333333', // -> #333333
+		'black-color-text': '#111111',
+		'bg-profile': '#00C0FF', // -> #00C0FF
```

Comment:
Your comment describes the same meaning of the color, it's completely useless. 👎